packages/core/src/Currency.ts

```diff
@@ -1,12 +1,46 @@
 import { z } from "zod";
 
 export const CurrencySchema = z.object({
+  /**
+   * The currency code, eg. "USD".
+   */
   code: z.string(),
+
+  /**
+   * The full currency name, eg. "US Dollars".
+   */
   name: z.string(),
```

Comment:
Since it's used for display - maybe worth clarifying if it should be the plural name or have a separate field for that? Maybe it doesn't matter.

Reply:
Done!